pom.xml

```diff
@@ -108,7 +108,7 @@
         <dependency>
             <groupId>org.ini4j</groupId>
             <artifactId>ini4j</artifactId>
-            <version>0.5.4</version>
+            <version>0.5.2</version>
```

Comment:
maybe needs some comment for future update attempts?

Reply:
Kept that in the Dependabot config.